herd/machModelChecker.ml

```diff
@@ -372,10 +376,12 @@ module Make
                       let evts_w_attr = E.EventSet.add e evts_w_attr in
                       let evts_map = StringMap.add attr evts_w_attr evts_map in
                       evts_map) attrs evts_map)
-                impl_pte_reads StringMap.empty in
+                pte_accesses StringMap.empty in
             I.add_sets m
               (StringMap.fold
-                 (fun k v l -> ("PTE" ^ k, lazy v) :: l) evts_map [])
+                 (fun k v l ->
+                   ("PTE" ^ k, lazy v) :: ("PTEMemAttr", lazy v) :: l)
```

Comment:
Hi @relokin, This code may attempt to perform the "PTEMemAttr" binding several tiimes, once per key `k`  present in `evts_map`.

Reply:
Thanks for the review @maranget and sorry for the delay in addressing your comment.

Very good point, I just pushed an update which I believe addresses this.